.gitignore

```diff
@@ -1,5 +1,7 @@
 .DS_Store
-.pyc
+*~
+*.svg
+*.pyc
```

Comment:
OK, I'd remove if not many people need. *~ files are temporary files generated by vim.